crates/bevy_pbr/src/prepass/mod.rs

```diff
@@ -104,14 +135,11 @@ where
                     .in_set(RenderSet::Prepare)
                     .after(bevy_render::view::prepare_windows),
             )
-            .add_system(queue_prepass_view_bind_group::<M>.in_set(RenderSet::Queue))
             .add_system(queue_prepass_material_meshes::<M>.in_set(RenderSet::Queue))
             .add_system(sort_phase_system::<Opaque3dPrepass>.in_set(RenderSet::PhaseSort))
             .add_system(sort_phase_system::<AlphaMask3dPrepass>.in_set(RenderSet::PhaseSort))
-            .init_resource::<PrepassPipeline<M>>()
             .init_resource::<DrawFunctions<Opaque3dPrepass>>()
             .init_resource::<DrawFunctions<AlphaMask3dPrepass>>()
-            .init_resource::<PrepassViewBindGroup>()
             .init_resource::<SpecializedMeshPipelines<PrepassPipeline<M>>>()
```

Comment:
Isn't this already added by the prepassPipelinePlugin?

Reply:
Yes, thank you for catching that. I fixed that.